understanding/22/focus-not-obscured-minimum.html

```diff
@@ -40,43 +40,43 @@ <h2>Intent of Focus Not Obscured</h2>
 
     </section>
 
-    <section id="benefits">
+<section id="benefits">
       <h2>Benefits of Focus Not Obscured</h2>
       
       <ul>
-         <li>Sighted users who rely on a keyboard interface to operate the page will be able to see the component which gets keyboard focus. Such users include those who rely on devices which use the keyboard interface, including speech input, sip-and-puff software, onscreen keyboards, scanning software, and a variety of assistive technologies and alternate keyboards.</li>
+         <li>Sighted users who rely on a keyboard interface to operate the page will be able to see the component which gets keyboard focus. Such users include those who rely on devices which use the keyboard interface, including speech input, sip-and-puff software, on-screen keyboards, scanning software, and a variety of assistive technologies and alternate keyboards.</li>
          <li>People with limited or low vision, who may primarily user a pointer for screen orientation and repositioning, nonetheless benefit from a visible indication of the current point of keyboard interaction, especially where magnification reduces the overall viewing portion of the screen.</li>
          <li>People with attention limitations, short term memory limitations, or limitations in executive processes benefit by being able to discover where the focus is located.</li>
       </ul>
       
    </section>
    
-   <section id="examples">
-      <h2>Examples of Focus Not Obscured</h2>
+<section id="examples">
+  <h2>Examples of Focus Not Obscured (Minimum)</h2>
 
-      <ul>
-         <li>A page has a sticky footer (attached to the bottom of the viewport). When tabbing down the page the focused item is not hidden by the footer.</li>
-         <li>A page has a large (30% wide) cookie approval dialogue. The dialogue is modal, preventing access to the other controls in the page until it has been dismissed.</li>
-         <li>A notification is implemented as a sticky header and the keyboard focus is moved to the notification. The notification is dismissed when it loses focus so it does not obscure any other controls.</li>
-      </ul>
+   <ul>
+      <li>A page has a sticky footer (attached to the bottom of the viewport). When tabbing down the page the focused item is not hidden by the footer because content in the viewport scrolls up to always display the item with keyboard focus using <a href="https://www.w3.org/TR/css-scroll-snap/#propdef-scroll-padding" rel="nofollow">scroll padding</a>.</li>
```

Comment:
```suggestion
      <li>A page has a sticky footer (attached to the bottom of the viewport). When tabbing down the page the focused item is not completely hidden by the footer because content in the viewport scrolls up to always display the item with keyboard focus using <a href="https://www.w3.org/TR/css-scroll-snap/#propdef-scroll-padding" rel="nofollow">scroll padding</a>.</li>
```